src/database/queries/getUser.js

```diff
@@ -0,0 +1,3 @@
+const connection = require('../connection.js');
+const getUser= (user_name)=> connection.query(`select * from users where user_name=$1`,[user_name]);
+module.exports =getUser;query
```

Comment:
@denaHS Hi! I think you have an extra `query` on the last line, after the export.